### PR TITLE
Improve explanation of config.yaml

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

**Comment by bmansheim** [comment]
@mcouthon A customer complained that there was no explanation of how to configure multi-network management with the config.yaml. Please enter how the agent networks section should look for each of the two scenarios that I added.

File: content/installation/installing-manager.md [hunk 1]
```diff
@@ -93,16 +93,34 @@ After you install Cloudify Manager, you can change the settings used by the inst
...
 * External REST communications over HTTPS
 * Local path replacement for remote resources with a URL
-* Management networks for Cloudify agents
+* Multi-network management
 * LDAP connection information
 * SSL communication settings
 
+#### Multiple Networks
+
+If a manager has a private IP (1.2.3.4) and two additional IPs (10.0.0.1 and 192.168.0.2) through which an agent can connect to it, each IP must be specified in the config.yaml. In the blueprint the agent can be configured to use one of these networks. If no IP address is specified in the blueprint, the agent connects to the private IP (1.2.3.4).
```
Comment (mcouthon) [inline content/installation/installing-manager.md]: No need to differentiate between private and additional IPs. They are all private IPs, just on different interfaces

File: content/installation/installing-manager.md [hunk 2]
```diff
@@ -93,16 +93,34 @@ After you install Cloudify Manager, you can change the settings used by the inst
...
+
+agent:
+  networks: {}
+  broker_port: 5671
+  min_workers: 2
+  max_workers: 5
+
+If the manager has a private IP (1.2.3.4) for its internal services and two additional IPs (10.0.0.1 and 192.168.0.2) for agents to communicate with it, the agent can be configured to use one of the additional IP addresses instead of the internal IP address.
```
Comment (mcouthon) [inline content/installation/installing-manager.md]: Aren't we repeating ourselves here?

File: content/installation/installing-manager.md [hunk 3]
```diff
@@ -93,16 +93,34 @@ After you install Cloudify Manager, you can change the settings used by the inst
...
-* Management networks for Cloudify agents
+* Multi-network management
 * LDAP connection information
 * SSL communication settings
 
+#### Multiple Networks
+
+If a manager has a private IP (1.2.3.4) and two additional IPs (10.0.0.1 and 192.168.0.2) through which an agent can connect to it, each IP must be specified in the config.yaml. In the blueprint the agent can be configured to use one of these networks. If no IP address is specified in the blueprint, the agent connects to the private IP (1.2.3.4).
+
+agent:
```
Comment (mcouthon) [inline content/installation/installing-manager.md]: Should be stylized as code

File: content/installation/installing-manager.md [hunk 4]
```diff
@@ -93,16 +93,34 @@ After you install Cloudify Manager, you can change the settings used by the inst
...
+  networks: {}
```
Comment (mcouthon) [inline content/installation/installing-manager.md]:
```
networks:
  network_a: 10.0.0.1
  network_b: 192.168.0.1
```

**Pull request merged**